tests/everflow/everflow_test_utilities.py

```diff
@@ -1094,8 +1157,7 @@ def copy_and_pad(pkt, asic_type, platform_asic, hwsku, multi_binding_acl=False):
                     padded = binascii.unhexlify("0" * 44) + bytes(padded)
         if asic_type in ["barefoot", "cisco-8000", "marvell-teralynx"] \
            or platform_asic == "broadcom-dnx" \
-           or hwsku in ["rd98DX35xx", "rd98DX35xx_cn9131"] \
-           or hwsku.startswith("Nokia-7215-A1"):
```

Comment:
Let's not change this logic because there could be HWSKU name that contains `Nokia-7215-A1`

Reply:
Didnt have a chance to verify the diff until now. This logic just wasnt in the internal branch I was working on. Ill revert this change and any others that arent from the test proper